hadoop-tools/hadoop-aws/src/test/java/org/apache/hadoop/fs/s3a/scale/AbstractSTestS3AHugeFiles.java

```diff
@@ -113,6 +113,16 @@ public void setup() throws Exception {
         DEFAULT_HUGE_FILESIZE);
   }
 
+  /**
+   * Test dir deletion is removed from test case teardown so the
+   * subsequent tests see the output.
+   * @throws IOException failure
+   */
+  @Override
+  protected void deleteTestDirInTeardown() throws IOException {
```

Comment:
So for the huge files test, we will have to delete manually via the S3 lifecycle rule as mentioned in the testting.doc? 
This is just to reduce the time taken by tests right?

Reply:
its because we can't have one parallel test run cleaning the entire bucket while another is active. they MUST work on subdirs only

Reply:
that is only for the huge file. the testPath still remains right?